### PR TITLE
rclone: 1.33 -> 1.35

### PR DESCRIPTION
**Pull request by danielfullmer**
###### Motivation for this change

Updated rclone to 1.35. Note that the go dependencies are included under the vendor/ directory of the source files. So, there is no need to include them in the deps.json file, as discussed in [go2nix#21](https://github.com/kamilchm/go2nix/issues/21)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/sync/rclone/default.nix [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
     owner = "ncw";
     repo = "rclone";
     rev = "v${version}";
-    sha256 = "00y48ww40x73xpdvkzfhllwvbh9a2ffmmkc6ri9343wvmb53laqk";
+    sha256 = "15dmppb7qgr3vg76dsv770l51lmsl8n8k3rvbnhhks5a2cz0kf2i";
   };
 
-  goDeps = ./deps.nix;
-
   meta = {
```
Comment (LnL7) [inline pkgs/applications/networking/sync/rclone/default.nix]: You could use `meta = with stdenv.lib; ...` to make it a little less verbose.

File: pkgs/applications/networking/sync/rclone/deps.nix [hunk 2]
```diff
@@ -1,219 +0,0 @@
-# This file was generated by go2nix.
```
Comment (LnL7) [inline pkgs/applications/networking/sync/rclone/deps.nix]: What happened to all the dependencies?

**Review by LnL7 — APPROVED** [review]

File: pkgs/applications/networking/sync/rclone/default.nix (hunk 1 above)
Comment (danielfullmer) [inline pkgs/applications/networking/sync/rclone/default.nix]: Updated

File: pkgs/applications/networking/sync/rclone/deps.nix (hunk 2 above)
Comment (danielfullmer) [inline pkgs/applications/networking/sync/rclone/deps.nix]: As far as I can tell, this file is no longer necessary. The rclone project pins and includes its own go dependencies under the `vendor` subdirectory. See https://github.com/ncw/rclone/tree/master/vendor
Moreover, go2nix has been updated to take the `vendor` sudirectory into account when computing deps.nix. See [go2nix#21](https://github.com/kamilchm/go2nix/issues/21) In fact, running go2nix on this project now produces an empty deps.nix file.

Comment (LnL7) [inline pkgs/applications/networking/sync/rclone/deps.nix]: Ah that explains it, was just curious.

**Pull request merged**